.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master, rel/* ]
```

Comment:
This will mean the CI only runs on PRs to master and rel/*, you might want it to _run_ on all PRs, but only _require_ it for master and rel (in your rulesets)